### DISCUSSION TITLE
>1000 Kin spends potential flexibility improvement

### DISCUSSION THREAD

**Discussion by samdowd** [kinfoundation/kin-rfcs#130] [_ Archive - KRE 3.1 Discussion]
Some discussion already on the new >1000 rule (https://github.com/kinecosystem/rewards-engine/discussions/111 by @ben4trader). Starting a new thread since I have a new idea, and I don't know GitHub Discussion etiquette.

It makes sense to me that 1 Kin spends are meaningless, so something should be done to reward apps whose users spend real amounts. But I wonder if "each transaction has to be X Kin" is the best way to look at it. Adding a rule like this encourages playing along with the system in ways that aren't in line with improving user experience, and it meaningfully restricts potential use cases.

*My understanding of the current proposal is that a valid user is one who makes three 1000+ Kin spends in a one month period.*

*My alternative proposal is that a valid user is one who spends a total of 3000+ Kin in a one month period.*

This has the same effect as far as total Kin spent. But it also allows apps to reach the number organically, in ways that are in line with the best version of their user experience. And it seems like it would be a trivial amount of extra math for the KRE processor to do, as well as being easier to understand for ecosystem partners trying to parse the KRE.

Maybe I overlooked something though. Happy to be proven wrong.

**Comment by samdowd** [comment]
Oh and I'm not entirely sure how the proposal defines median spend, but my proposal is to define median spend as the median total-spend-of-a-user over the one month period. Probably keeping the same logic of only counting Active Users (spent 3000+) as part of this calculation, like the current proposal does.

**Reply by skyf0xx** [reply]
@samdowd Interesting proposal. Let me think this through as to how it would play out and what the advantages/ disadvantages would be to this.
